core/src/UVMap.cpp

```diff
@@ -70,6 +76,9 @@ void UVMap::ratio(double w, double h)
     ratio_.height = h;
     ratio_.aspect = w / h;
 }
+
+void UVMap::clear_map() { map_.clear(); }
+
```

Comment:
We don't need this in the public interface. Since `AlignToAxis` is a static member function, you should be able to just call `samples_uvs.map_.clear()` directly.